sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/ValueModifier.java

```diff
@@ -26,17 +26,21 @@
 
   Operation operation();
 
-  Value<?> value();
+  Term<?> value();
 
   static ValueModifier set(String columnName, Object value) {
-    return of(Target.column(columnName), Operation.SET, Value.of(value));
+    return set(columnName, Term.of(value));
+  }
+
+  static ValueModifier set(String columnName, Term<?> value) {
```

Comment:
Should this hten be `TermModifier` for the sake of consistency? (not sure myself)

Reply:
I had it that way initially, but then I realized that it's more a reference to the `VALUES` keyword in the CQL language, rather than the concept of a bound or literal value. Similar to how `QueryBuilderImpl.value()` didn't change either.

Reply:
Ok that does make sense, thanks.